runtime/service.go

```diff
@@ -738,3 +754,32 @@ func packBundle(path string, options *ptypes.Any) (*ptypes.Any, error) {
 	}
 	return ptypes.MarshalAny(extraData)
 }
+
+// setupStubDrive will create a flat file with the drive ID as its contents.
+// The file is padded with null bytes to a minimum size of minFCDriveImageSize.
+// This is due to Firecracker minimum drive image size limits.
+func setupStubDrive(builder firecracker.DrivesBuilder, dir, driveID string) (firecracker.DrivesBuilder, error) {
+	path := filepath.Join(dir, driveID)
+	f, err := os.Create(path)
```

Comment:
What are the permissions on this file? Something like `0600` seems best in this case

Reply:
(Though we should consider what impact that might have on the jailer, I'm not sure off the top of my head if the jailer supports running firecracker as a non-root user)

Reply:
permissions are `0666` with `Create` and jailer requires root

Reply:
Right, so I think we should set the permissions to 0600 since nobody but root needs to read these then